zum/requests/core.py

```diff
@@ -2,12 +2,44 @@
 Module for the core requests logic of zum.
 """
 
-from typing import Any, Dict, List
+from json import load
+from pathlib import Path
+from typing import Any, Dict, List, NoReturn, Union
 
+from zum.requests.errors import InvalidRequestBodyFileError
 from zum.requests.helpers import reduce_arguments
 from zum.requests.models import Request
 
 
+def load_request_body(arguments: List[str]) -> Union[Dict[str, Any], NoReturn]:
+    """
+    Parse the JSON request body into a dictionary and return it if the
+    specified file exists.
+    """
+    if len(arguments) >= 1:
+        body_file: Path = Path(arguments.pop())
+        if body_file.exists() and body_file.is_file():
+            with body_file.open("r") as body_contents:
+                return load(body_contents)
+
+        else:
+            if not body_file.exists():
+                raise InvalidRequestBodyFileError(
+                    f"Request body file located at '{body_file.absolute()}' does"
+                    + " not exist!"
+                )
+
+            if not body_file.is_file():
+                raise InvalidRequestBodyFileError(
+                    f"Request body file located at '{body_file.absolute()}' is not"
+                    + " a file!"
```

Comment:
I would loose the `+` operator (as the parentheses will concatenate the strings) and leave the space on the upper line, for consistency's sake